tests/compiletest.rs

```diff
@@ -250,6 +269,21 @@ fn main() -> Result<()> {
     Ok(())
 }
 
+fn run_dep_mode(target: String) -> Result<()> {
+    let files = std::env::args().skip(2);
+    for path in files {
```

Comment:
I would expect `./miri run-dep file.rs foo bar` to be interpreted as running `file.rs` with arguments `foo bar`. That's what `./miri run` does. This seems to interpret it as running `foo` and `bar` as well?

Reply:
😆 whoops.